lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -45,10 +46,11 @@ export class ServerRequestParameters {
    * @param redirectUri
    * @param state
    */
-  constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string ) {
+  constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string, popToken?: string ) {
```

Comment:
is popToken the thing that is being passed here?

Reply:
Not sure what you mean. `popToken` will be the public key hash that we generate and send to STS.